spacy/tests/test_cli.py

```diff
@@ -855,3 +857,126 @@ def test_span_length_freq_dist_output_must_be_correct():
     span_freqs = _get_spans_length_freq_dist(sample_span_lengths, threshold)
     assert sum(span_freqs.values()) >= threshold
     assert list(span_freqs.keys()) == [3, 1, 4, 5, 2]
+
+
+def test_cli_find_threshold(capsys):
+    thresholds = numpy.linspace(0, 1, 10)
+
+    def make_examples(_nlp: Language) -> List[Example]:
```

Comment:
```suggestion
    def make_examples(nlp: Language) -> List[Example]:
```

Reply:
@rmitsch : can you apply this suggestion + alter the var name in the body as well?